src/main/java/Duke.java

```diff
@@ -1,10 +1,115 @@
+import java.util.ArrayList;
+import java.util.Scanner;
 public class Duke {
-    public static void main(String[] args) {
+    private static ArrayList<Task> tasks = new ArrayList<>();
+
+    public static void printLine(){
+        System.out.println("-----------------------------------");
+    }
+
+    public static void greet(){
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+        printLine();
+        System.out.println("Hello, I'm Duke");
+        System.out.println("What can I do for you?");
+        printLine();
+    }
+
+    public static void bye(){
+        printLine();
+        System.out.println("Bye. Hope to see you soon!");
+        printLine();
+    }
+
+    public static void addToList(String input){
+        if(input.startsWith("deadline")) {
+            addDeadline(input);
+        } else if(input.startsWith("event")){
+            addEvent(input);
+        } else if(input.startsWith("todo")){
+            addTodo(input);
+        } else {
+            tasks.add(new Todo(input));
+        }
+
+        printLine();
+        System.out.println("Got it. I've added this task:");
+        System.out.println(tasks.get(tasks.size() - 1));
+        System.out.println("Now you have " + tasks.size() + " tasks in the list.");
+        printLine();
+    }
+
+    private static void addTodo(String input) {
+        input = input.split("todo")[1].trim();
+        tasks.add(new Todo(input));
+    }
+
+    private static void addEvent(String input) {
+        input = input.split("event")[1].trim();
+        String description = input.split("/at")[0].trim();
+        String time = input.split("/at")[1].trim();
+        tasks.add(new Event(description, time));
+    }
+
+    private static void addDeadline(String input) {
+        input = input.split("deadline")[1].trim();
```

Comment:
Consider assigning `input.split("event")[1].trim()` to a variable (for example `deadlineDetails`) that represents the purpose of this argument?